Cubical/Categories/Constructions/EssentialImage.agda

```diff
@@ -0,0 +1,67 @@
+{-
+
+The Essential Image of Functor
+
+-}
+{-# OPTIONS --safe #-}
+module Cubical.Categories.Constructions.EssentialImage where
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Data.Sigma
+open import Cubical.HITs.PropositionalTruncation as Prop
```

Comment:
How about using `PT` instead of `Prop`?

Reply:
Is `PropTrunc` OK? Though it's a bit long...

Reply:
Is `PropTrunc` OK? Though it's a bit long...

Reply:
Sure, that's fine! I just don't like calling it just Prop